pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -45,6 +45,25 @@ var (
 		options.FrontProxyClientCertAndKeyName,
 	}
 
+	emptyByteSlice                 = make([]byte, 0)
+	externalEtcdCertSpecialization = map[string]func(*CommandInitOption) ([]byte, []byte, error){
+		options.EtcdCaCertAndKeyName: func(option *CommandInitOption) (cert, key []byte, err error) {
+			cert, err = os.ReadFile(option.ExternalEtcdCACertPath)
```

Comment:
I mistakenly thought the err here was not handled, but in fact, it was thrown into `runInit` for processing. So, it is ok~